packages/cli/src/commands/generate/dbAuth/__tests__/dbAuth.test.js

```diff
@@ -1,8 +1,4 @@
-global.__dirname = __dirname
-
-jest.mock('fs')
-
-import fs from 'fs'
```

Comment:
Seems like we need the `fs` stuff back here: https://github.com/redwoodjs/redwood/actions/runs/3692167967/jobs/6250798520